lib/_http_outgoing.js

```diff
@@ -1005,6 +1019,120 @@ function connectionCorkNT(conn) {
   conn.uncork();
 }
 
+OutgoingMessage.prototype.sendFile = function sendFile(pathOrFd, optsOrCallback, callbackOrNil) {
```

Comment:
@mcollina I don't remember know what we prefer in terms of errors for asynchronous methods? I assume arg validation should throw synchronously and everything else asynchronously?

Reply:
I don't think we have a common rule. I would throw in this case.